client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -197,110 +207,118 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
+                            }}
+                            selectedValue={getIn(
+                              values,
+                              `sampleSizes[${contest.id}][key]`
+                            )}
+                            disabled={locked}
+                          >
+                            {sampleSizeOptions[contest.id].map(
+                              (option: ISampleSizeOption) => {
+                                return option.key === 'custom' ? (
+                                  <Radio value="custom" key={option.key}>
+                                    Enter your own sample size (not recommended)
+                                  </Radio>
+                                ) : (
+                                  <Radio value={option.key} key={option.key}>
+                                    {option.key === 'asn'
+                                      ? 'BRAVO Average Sample Number: '
+                                      : ''}
+                                    {`${option.size} samples`}
+                                    {option.prob
+                                      ? ` (${percentFormatter.format(
+                                          option.prob
+                                        )} chance of reaching risk limit and completing the audit in one round)`
+                                      : ''}
+                                  </Radio>
+                                )
+                              }
                             )}
-                          />
-                        )}
-                      </FormSectionDescription>
-                    </ElevatedCard>
-                  )
-                })}
-              </FormSection>
-            )}
-            <FormButtonBar>
-              <FormButton onClick={prevStage.activate}>Back</FormButton>
-              <FormButton
-                intent="primary"
-                disabled={
-                  locked ||
-                  !isSetupComplete(jurisdictions, contests, auditSettings)
-                }
-                onClick={handleSubmit}
-              >
-                Launch Audit
-              </FormButton>
-            </FormButtonBar>
-          </Form>
-        )}
-      </Formik>
+                          </RadioGroup>
+                          {currentOption && currentOption.key === 'custom' && (
+                            <Field
+                              component={FormField}
+                              name={`sampleSizes[${contest.id}][size]`}
+                              type="text"
+                              validate={testNumber(
+                                Number(contest.totalBallotsCast),
+                                `Must be less than or equal to: ${contest.totalBallotsCast} (the total number of ballots in this targeted contest)`
```

Comment:
this validation probably needs to change for batch audits

Reply:
on the backend we check that it's no greater than the total number of batches, so you could also just remove the validation and let the backend handle it if that's easier

Reply:
@jonahkagan In general I think it's safest to mirror backend validation on the frontend as much as possible, so users don't have to do things over again if they mess up. Much more friendly to let them know right away.

@mcchilders What's your take on this one?

Reply:
@MorganLove yes, let's mirror if possible.

Reply:
Okay so @jonahkagan what's the exact math that's used on the backend here? Do you mean the total number of ballots in all the batches?

Reply:
Omg I just realized we don't have validation of custom sample sizes on the backend after all (for any audit type). I got confused cuz there's so much other validation.

Reply:
The sample size for batch audits is the number of batches to be sampled, so we should validate that it's no greater than the total number of batches across the jurisdictions included in the audit.

Reply:
Just created #740 to implement the backend validation

Reply:
@jonahkagan just changed the validation of the custom sample size for batch comparison audits to just check that it's a number, since evidently the math doesn't require that it have the batch number validation. From what I can tell adding in that validation will add a significant amount of complexity, so for MVP we can do without.